### PR TITLE
Added CORS, Issue DTO and Issue Endpoints

### PR DESCRIPTION
**Pull request by mukul-mehta**
# Changes proposed through this PR
  
- Added Flask-CORS
- Created dto for issue.
- Created endpoints for issues.

### Needs to be tested with DB model.

### CODE REVIEW

**Comment by thealphadollar** [comment]
@mukul-mehta Please add the new endpoint I just added in the spec-sheet. I'll review this tomorrow.

File: Pipfile [hunk 1]
```diff
@@ -38,6 +38,9 @@ pymysql = "*"
 flask-wtf = "*"
 requests = "*"
 sendgrid = "*"
+python-dotenv = "*"
```
Comment (thealphadollar) [inline Pipfile]: remove dotenv

File: Pipfile [hunk 2]
```diff
@@ -38,6 +38,9 @@ pymysql = "*"
...
+flask-cors = "*"
+autopep8 = "*"
```
Comment (thealphadollar) [inline Pipfile]: remove autopep8

File: app/main/util/dto.py [hunk 3]
```diff
@@ -87,3 +87,12 @@ class TagDto:
 		'tag_id': fields.Integer(required=True, description="id of the concerned tag"),	
 		'value': fields.Integer(required=True, description="priority level of the tag")
 	})
+
+class IssueDto:
+	api = Namespace('issue', description="for issue related operations")
+	issue = api.model('tag', {
+		'cover' : fields.Integer(required=True, description="Cover image of the concerned issue"),
```
Comment (thealphadollar) [inline app/main/util/dto.py]: You also need to send back, `id`, `issue_tag`. It is mentioned in the specifications sheet.

Comment (mukul-mehta) [reply app/main/util/dto.py]: @grapheo12 and I discussed that ID and issue_tag are autogenerated and don't have to be present in the API model.

**Pull request merged**